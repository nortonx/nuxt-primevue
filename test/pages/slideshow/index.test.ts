import { shallowMount } from "@vue/test-utils";
import Slideshow from "@/pages/slideshow/index.vue";
import { describe, it, expect } from "vitest";

describe("Slideshow page", () => {
  it("should render correctly and match snapshot", () => {
    const wrapper = shallowMount(Slideshow);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
