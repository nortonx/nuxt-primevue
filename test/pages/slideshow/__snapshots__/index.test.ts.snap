// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Slideshow page > should render correctly and match snapshot 1`] = `
"<div class="p-4 md:px-6 lg:px-8">
  <div class="grid">
    <div class="col-12">
      <h1 class="text-4xl font-bold text-center mb-4">Slideshow</h1>
      <p class="text-center text-800"> This is a simple slideshow created using PrimeVue components. </p>
      <tab-view-stub></tab-view-stub>
    </div>
  </div>
</div>"
`;
